packages/@vuepress/theme-default/components/Page.vue

```diff
@@ -169,6 +163,29 @@ function resolveNext (page, items) {
   return find(page, items, 1)
 }
 
+function resolvePageLink (linkType, { $themeConfig, $page, $route, $site, sidebarItems }) {
+  const resolveLink = linkType === LINK_TYPES.NEXT ? resolveNext : resolvePrev
```

Comment:
This can be simplified if we add the resolve functions to the `LINK_TYPES` dictionary. WDYT?

```js
const LINK_TYPES = {
  NEXT: {
    name: 'next',
    resolveLink: resolveNext,
    themeLinkConfig: $themeConfig.nextLinks,
    pageLinkConfig: $page.frontmatter.next
  },
  PREV: 'prev',
  resolveLink: resolvePrev,
  themeLinkConfig: $themeConfig.prevLinks,
  pageLinkConfig: $page.frontmatter.prev
}

// ...

const { resolveLink, themeLinkConfig, pageLinkConfig } = linkType
```

Reply:
I did an update based on your comment. WDYT?

Reply:
Great, like the pure approach 🙂